src/pages/Pool/PositionPage.tsx

```diff
@@ -49,6 +50,22 @@ import { calculateGasMargin } from '../../utils/calculateGasMargin'
 import { ExplorerDataType, getExplorerLink } from '../../utils/getExplorerLink'
 import { LoadingRows } from './styleds'
 
+const getTokenLink = (chainId: SupportedChainId, address: string) => {
+  const chainName = CHAIN_ID_TO_NAME[chainId]
+  if (chainName) {
+    return `${window.location.hostname}/${chainName}/${address}`
+  } else {
+    return getExplorerLink(chainId, address, ExplorerDataType.TOKEN)
+  }
+}
+
+const CHAIN_ID_TO_NAME: Partial<Record<SupportedChainId, string>> = {
```

Comment:
@cartcrom I notice we have something similar in graphql data utils. Do you think it makes sense to use that, or create a separate mapping for just the frontend somewhere?

Reply:
i've used existing maps without needing to create a new one - if the existing ones work for your use case i'd definitely suggest re-using

Reply:
okay i'll switch to using that